LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -26,7 +26,7 @@ public class TherapySettingsViewModel: ObservableObject {
     let sensitivityOverridesEnabled: Bool
     public var prescription: Prescription?
     
-    let preferredGlucoseUnit: HKUnit
+    @Published public var preferredGlucoseUnit: HKUnit
```

Comment:
This seems like a good candidate for inclusion in the swiftui environment. WDYT?

Reply:
I had suggested this as well.

Reply:
I'm for it.